vagrant/Makefile

```diff
@@ -73,12 +74,34 @@ apply-service-monitors:
 apply-avalanche:
 	kubectl apply -f ${k8s_path}/avalanche.yaml
 
+# This applies prometheus remote write rule to forward all avalanche metrics to receiver-mock
+# Since this removes the generated values.local.yaml after it's applied subsequent 'make helm-upgrade'
+# will revert this change.
+apply-avalanche-remote-write: _apply-avalanche-remote-write-to-local-values helm-upgrade clean-local-values-yaml
+
+clean-local-values-yaml:
+	rm ${local_values_file}
+
+_apply-avalanche-remote-write-to-local-values: generate-local-config
+	yq w -s ${vagrant_scripts_dir}yq/update_remote_write_for_avalanche.yaml ../deploy/helm/sumologic/values.yaml | \
```

Comment:
Where is this (`update_remote_write_for_avalanche.yaml`) file? 🤔

Reply:
Ups, added in https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1025/commits/acac7d58f006611b08c621239b93c6ae138cf4f2